test/check-machines-lifecycle

```diff
@@ -60,9 +60,14 @@ class TestMachinesLifecycle(VirtualMachinesCase):
         user = self.createUser(user_group='libvirt')
         self._testBasic(user, False)
 
-    def testBasicWheelUserUnprivileged(self):
+    def testBasicWheelUser(self):
         user = self.createUser(user_group='wheel')
+
+        # Check NON-PRIVILEGED user won't see any VMs even though he is in wheel group
         self._testBasic(user, False, True)
+        self.machine.execute("virsh destroy subVmTest1 && virsh undefine subVmTest1")
+        # Check wheel group PRIVILEGED user will see VMs and can do operations on them
+        self._testBasic(user, True, False)
```

Comment:
That makes sense, thanks! Would be even nicer to name the boolean arguments, for easier interpretation. I.e. `superuser=True`